torchmetrics/functional/regression/mean_absolute_percentage_error.py

```diff
@@ -13,21 +13,20 @@
 # limitations under the License.
 from typing import Tuple
 
-import numpy as np
 import torch
-from torch import Tensor, tensor
-from torch._C import dtype
+from torch import Tensor
 
 from torchmetrics.utilities.checks import _check_same_shape
 
 
-def _mean_absolute_percentage_error_update(preds: torch.Tensor, target: torch.Tensor, eps: torch.Tensor) -> Tuple[Tensor, int]:
+def _mean_absolute_percentage_error_update(
+    preds: torch.Tensor, target: torch.Tensor, epsilon: torch.Tensor
```

Comment:
maybe consistently use `Tensor` vs `torch.Tensor` here? Shorter is probably better.

Reply:
I have fixed this in the latest commit! Thanks!